### ISSUE TITLE
repsvtr english support

### ISSUE THREAD

**Issue by AnanasPizzaMigliore** [Topdu/OpenOCR#162]
I tried to finetune the repsvtr_ch with an English dataset, but the accuracy was keeping dropping. So, i tried to train repsvtr_en from scratch. the accuracy is still super low. May I ask if author can share the pretrained model of repsvtr_en. I would appreciate a lot.

**Comment by Topdu** [comment]
We sincerely apologize, but we only have the Chinese version available; there is no English version.
